packages/@o3r/testing/schematics/ng-add/fixture/index.ts

```diff
@@ -8,10 +8,9 @@ import * as ts from 'typescript';
  * Add fixture configuration
  * @param options @see RuleFactory.options
  * @param options.projectName
- * @param registerUnitTestFixturePaths should add path mapping for unit test fixtures
  * @param options.testingFramework
  */
-export function updateFixtureConfig(options: { projectName?: string | null | undefined; testingFramework?: string | null | undefined }, registerUnitTestFixturePaths: boolean): Rule {
+export function updateFixtureConfig(options: { projectName?: string | null | undefined; testingFramework?: string | null | undefined }): Rule {
```

Comment:
we can't do breaking change

Reply:
we cannot do a breaking change if the method is exposed from the package and here it is not the case. 
![image](https://github.com/AmadeusITGroup/otter/assets/86055112/db2acf5e-5ad8-43fa-87e0-8165777b4aa8)